railties/lib/rails/generators/rails/app/templates/app/views/layouts/application.html.erb.tt

```diff
@@ -3,9 +3,6 @@
   <head>
     <title><%= camelized %></title>
     <%%= csrf_meta_tags %>
-    <%- unless options[:skip_action_cable] -%>
-    <%%= action_cable_meta_tag %>
```

Comment:
I'd like to keep this. It's going to be very common for large apps to run WebSocket servers separately from normal web actions. When you do that, you need a special URL/port setup. We should be ready for that. But we don't have to include it in the HTML by default. We can just assume a default of /cable. 

> On Feb 14, 2016, at 03:11, Matthew Draper <notifications@github.com> wrote:
> 
> In railties/lib/rails/generators/rails/app/templates/app/views/layouts/application.html.erb.tt:
> 
> > @@ -3,9 +3,6 @@
> >    <head>
> >      <title><%= camelized %></title>
> >      <%%= csrf_meta_tags %>
> > -    <%- unless options[:skip_action_cable] -%>
> > -    <%%= action_cable_meta_tag %>
> +1 let's remove the method, and just document how to build the meta tag "manually" if you need to specify a custom mount point.
> 
> —
> Reply to this email directly or view it on GitHub.
>